README.md

```diff
@@ -230,6 +230,10 @@ postgres=# select * from t;
 > cargo neon stop
 ```
 
+#### Handling build failures
+
+If you encounter build failures during the initialization steps, it's best to stop everything (`cargo neon stop`) and remove the `.neon` directory.
```

Comment:
I'm not sure what this sentence means. If there is build failures then those should be before any binary starts, no? Then there is also nothing to stop, and nothing has been written to `.neon`. If there is a flaw in my understanding, please point it out.

Reply:
I spent a good time investigating why this doesn't start on Silicon Mac when the curl IPv6 doesn't work. Even when I patched the lib in, it did not start. When I removed the `.neon` directory and started over after fixing the issues, it worked straight away.

There's something in the code which does not like starting on a half-started/initialized `.neon` directory, which broke somehow. Which I believe is not worth investigating, because at this point just removing it and starting over is easier. There's no data in it, yet.

Will see if I can come up with a better description.

Reply:
Maybe remove the word "build"? At least I assume compilation when I read "build". Is it a postgres internal term for initdb/database cluster creation?

Reply:
Thank you, looks good now. Resolving.